### PR TITLE
 Update modules in pipeline template with new versions syntax

### PR DESCRIPTION
**Pull request by drpatelh**
Porting fixes from https://github.com/nf-core/modules/pull/780

### CODE REVIEW

**Review by JoseEspinosa — APPROVED** [review]
:rocket: 
Just made a small question out of curiosity but it is ready to merge!

**Comment by JoseEspinosa** [comment]
I just saw the error, we need to include `getProcessName` on the modules, sorry  I didn't spot it before approving. Is it even possible to undo an approval?

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
include `getProcessName` in the modules

**Review by JoseEspinosa — APPROVED** [review]
I just commit the change and now test are passing, feel free to merge @drpatelh :candy:

File: nf_core/pipeline-template/modules/nf-core/modules/custom/dumpsoftwareversions/main.nf [hunk 1]
```diff
@@ -22,9 +22,9 @@ process CUSTOM_DUMPSOFTWAREVERSIONS {
     path versions
 
     output:
-    path 'software_versions.yml'    , emit: yml
-    path 'software_versions_mqc.yml', emit: mqc_yml
-    path 'versions.yml'             , emit: versions
+    path "software_versions.yml"    , emit: yml
+    path "software_versions_mqc.yml", emit: mqc_yml
+    path "versions.yml"             , emit: versions
```
[the remark this answers is not in the document]
Comment (drpatelh) [reply nf_core/pipeline-template/modules/nf-core/modules/custom/dumpsoftwareversions/main.nf:27]: I think they were auto-formatted somehow 👀

**Pull request merged**